Central Command&ert Ships

WhiteLightningTwister opened the pull request:
<!--
This is a semi-strict format, you can add/remove sections as needed but the order/format should be kept the same
Remove these comments before submitting
-->

# Description

<!--
Explain this PR in as much detail as applicable

Some example prompts to consider:
How might this affect the game? The codebase?
What might be some alternatives to this?
How/Who does this benefit/hurt [the game/codebase]?
-->

This PR looks to refresh central command likewise add additional spawn able grids for admins to use which are more complete then before. Given my previous attempts I've gotten a lot better at making ships, likewise have properly tested if they work on spawn.

I have made mistakes in the past when it comes to these but i have taken extra steps to make sure these are properly functional. Likewise the new central command will allow admins to facilitate things at CC. Likewise allow for more expression from players at the end of the shift.

Central command gets to have fully flushed out departments likewise offices for those departments allowing players to settle down and relax after a stressful or boring shift.

---

# TODO

<!--
A list of everything you have to do before this PR is "complete"
You probably won't have to complete everything before merging but it's good to leave future references
-->

- [X] Tasks Completed.
- [X] Waiting for approval from headmins.
- [X] Items reviewed
- [X] CC 2.0
- [X] Death Squad Vessel 
- [X] CC-Cargoship
- [X] CC-Inspectionship
- [X] CC-Personalshuttle
- [X] CC-Prisonertransport
- [X] CC-Yacht
- [X] CC-Relaystation
- [X] CC-AIprobe  
- [X] ERT-Troopshuttle
- [X] ERT-Fighter
- [X] ERT-Superiorityfighter
- [X] ERT-Floatinghospital
- [X] ERT-Emergencycargoshuttle
- [X] Removed old vessels that outdated likewise not properly made.

---

<!--
This is default collapsed, readers click to expand it and see all your media
The PR media section can get very large at times, so this is a good way to keep it clean
The title is written using HTML tags
The title must be within the <summary> tags or you won't see it
-->

<details><summary><h1>New Central Command</h1></summary>
<p>

<img width="3904" height="4416" alt="CentComm-0" src="https://github.com/user-attachments/assets/ec28b16d-4b57-44e3-b590-7aec0f16a970" />

</p>
</details>


<details><summary><h1>Central Command Ships</h1></summary>
<p>


<img width="375" height="398" alt="CCpersonalshuttleradar" src="https://github.com/user-attachments/assets/a6b3a80d-3162-4947-95ae-fe48f09a55d5" />

<img width="724" height="711" alt="CCpersonalshuttle" src="https://github.com/user-attachments/assets/3415a790-c9d3-4037-a4f0-1667a17b089b" />

<img width="433" height="431" alt="CCprisonertransportradar" src="https://github.com/user-attachments/assets/28d63df7-fabe-469a-98be-f0643e18e223" />

<img width="612" height="777" alt="CCprisonertransport" src="https://github.com/user-attachments/assets/d7614887-c485-4400-ac48-9fdb9bddeb70" />

<img width="420" height="399" alt="CCcargoshipradar" src="https://github.com/user-attachments/assets/7368f92a-5923-4206-b27f-8028cffa0748" />

<img width="474" height="729" alt="CCcargoship" src="https://github.com/user-attachments/assets/766ff4e5-290f-412f-9e36-b0fa8a1dd0b5" />

<img width="471" height="464" alt="CCproberadar" src="https://github.com/user-attachments/assets/7af311c5-3c6d-4899-8eb8-2070223bf9a5" />

<img width="1268" height="743" alt="CCprobe" src="https://github.com/user-attachments/assets/17b44151-2498-4c73-b37c-5e11cfa34d8a" />

<img width="405" height="407" alt="CCrelaystationRadar" src="https://github.com/user-attachments/assets/c1204e2a-d362-4fb9-bc3f-262b8f6178f0" />

<img width="898" height="852" alt="CCrelaystation" src="https://github.com/user-attachments/assets/b19bfa95-f6dd-4e58-9f94-a89ef31b872a" />

<img width="501" height="507" alt="CCyachtRadar" src="https://github.com/user-attachments/assets/42c8db51-1f3b-4b86-b101-b41608bee4d7" />

<img width="699" height="720" alt="CCyacht" src="https://github.com/user-attachments/assets/b4e6d762-9aab-4a75-b6a0-d4967b0e2b5b" />

<img width="629" height="614" alt="Deathsquadradar" src="https://github.com/user-attachments/assets/44adc0f8-b17d-4438-8c86-2d37366d040a" />

<img width="995" height="748" alt="Deathsquad1" src="https://github.com/user-attachments/assets/53fca5cf-0134-4154-af12-cb0c5d926426" />

<img width="984" height="859" alt="Deathsquad2" src="https://github.com/user-attachments/assets/dcfc0b06-1a6d-4331-8e7a-1174fd48ad12" />


</p>
</details>


<details><summary><h1>ERT Vessels</h1></summary>
<p>

<img width="337" height="346" alt="ERTbettleradar" src="https://github.com/user-attachments/assets/6ef98830-d8cb-4380-a93f-903337174f01" />

<img width="498" height="724" alt="ERTBettle" src="https://github.com/user-attachments/assets/838f80b0-c8ed-4c78-b9d4-96f61b25d87e" />

<img width="280" height="268" alt="ERTfighterRadar" src="https://github.com/user-attachments/assets/4d34916b-2e32-4064-84f4-371b4e3d5513" />

<img width="739" height="724" alt="ERTfighter" src="https://github.com/user-attachments/assets/8de8e96e-e74a-48af-b21f-602b264ce934" />

<img width="372" height="350" alt="ERTfloatinghospitalradar" src="https://github.com/user-attachments/assets/b90dd986-779a-4f7f-9855-8869f671e700" />

<img width="561" height="769" alt="ERTfloatinghospital" src="https://github.com/user-attachments/assets/afccf74a-e53b-42bd-9762-2b9864b5d8dc" />

<img width="486" height="791" alt="ERTpelican" src="https://github.com/user-attachments/assets/d101de6f-8cdc-4169-878d-a4d768d50b29" />

<img width="360" height="371" alt="ERTpelicanRadar" src="https://github.com/user-attachments/assets/7005942a-91bc-416d-84c2-52af05808e45" />

<img width="354" height="384" alt="ERTsuperirorityfighterradar" src="https://github.com/user-attachments/assets/be35cfd9-33e2-4824-8f62-3c9c623277a2" />

<img width="585" height="758" alt="ERTsuperirorityfighter" src="https://github.com/user-attachments/assets/69093e39-7878-4541-9839-31347778ba90" />

</p>
</details>

---

# Changelog

<!--
You can add an author after the `:cl:` to change the name that appears in the changelog (ex: `:cl: Death`)
Leaving it blank will default to your GitHub display name
This includes all available types for the changelog
-->

:cl:
- add: Added a bunch of new grids that admins can use.
- tweak: Central Command 2.0
- remove: outdated ships.

WhiteLightningTwister committed:
Removing outdated or unused items

These ships were formalyl only really used by me likewise have minor inperfection or don't correctly work on start. So these are getting removed given other folders have been added and this one being created for ERT & CC. So just consolidating this.

WhiteLightningTwister committed:
New CC & ERT Admin spawns

New ships and installations which admins can spawn to use for ingame events or as a response to ungoing things in the round.

WhiteLightningTwister committed:
New CC

Entirely rebuilt. Tested if EVAC or EVAC pods dock. They appear to do so, the station is also functional.

WhiteLightningTwister committed:
Fixed invalids

Thanks to dunrab. He fixed my invalids and was able to get me a media image. Thanks!

WhiteLightningTwister committed:
Spawn issue + Additons

I had saved the things as mapping instead of grid, i had fixed them. Likewise made some additonal such as the ERT floating and CC relay satlite.

WhiteLightningTwister committed:
Removed Decals

Was requested to remove some decals at the EVAC area.

WhiteLightningTwister committed:
Merge branch 'Floof-Station:master' into CentralCommand&ERTShips

WhiteLightningTwister committed:
Merge branch 'Floof-Station:master' into CentralCommand&ERTShips

Dunrab commented on Resources/Maps/Floof/CentralCommand/CCStation/CCRelaysatellite:
need to remove this since its  duplicate of CCRelaysatellite.yml

WhiteLightningTwister replied:
Yeah, i'll be working on finishing this up in the next day or two. Then resubmitting everything.

WhiteLightningTwister committed:
Incorrect formatting

This was removed due to it being incorrect formatting likewise an accident copy of the other file.

WhiteLightningTwister committed:
Rename requested

This is being removed to make it more clear this a central command cargo shuttle.

WhiteLightningTwister committed:
Rename of cargo shuttle

The new renamed cargo shuttle to help identify it better.

WhiteLightningTwister committed:
Requested changes

These files were changed at the request of the review dunrab, in the way they had requested which was listed on the discord.

WhiteLightningTwister committed:
Merge branch 'Floof-Station:master' into CentralCommand&ERTShips

WhiteLightningTwister committed:
Fixed minor errors

made some corrections at Dunrab's request.

Dunrab reviewed (approved):
Requested changes were made and tests passed. New centcomm will be a nice addition.
